store/copr/coprocessor.go

```diff
@@ -954,6 +954,20 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 		data := make([]byte, len(cacheValue.Data))
 		copy(data, cacheValue.Data)
 		resp.pbResp.Data = data
+		if worker.req.Paging {
+			var start, end []byte
+			if cacheValue.PageStart != nil {
+				start = append([]byte{}, cacheValue.PageStart...)
```

Comment:
Why it is not `start = append(start, ...`? Will golang assign `start = cacheValue.PageStart` directly?

Reply:
`start = append(start, ...)` can be used here, it's a deep copy with (maybe several times) allocation.
I update the code here.